lib/auto-migrations/strategies/alter.js

```diff
@@ -121,7 +121,7 @@ module.exports = function alterStrategy(orm, cb) {
             }
 
             var lastRecord = _.last(backupRecords);
-            var primaryKeyColumnName = WLModel.schema[primaryKeyAttrName].columnName;
+            var primaryKeyColumnName = WLModel.schema[primaryKeyAttrName].columnName || primaryKeyAttrName;
```

Comment:
This should always be set at this point -- i.e. waterline-schema should make sure it exists.  So instead, maybe:

```
assert(WLModel.schema[primaryKeyAttrName].columnName, 'columnName should always be set at this point, but for the primary key attribute, it is not.');
```